packages/client/mutations/RemoveReflectTemplatePromptMutation.ts

```diff
@@ -49,10 +50,6 @@ const RemoveReflectTemplatePromptMutation = (
       const payload = store.getRootField('removeReflectTemplatePrompt')
       if (!payload) return
       removeReflectTemplatePromptTeamUpdater(payload as any, {atmosphere, store})
-    },
-    optimisticUpdater: (store) => {
```

Comment:
This `optimisticUpdater` was causing a console err of a store update within a store update. After removing it, I've tested removing a prompt on a 3G network and there doesn't seem to be any difference in the responsiveness